documentation/modules/metrics/ref_metrics-dashboards.adoc

```diff
@@ -0,0 +1,188 @@
+
+// This assembly is included in the following assemblies:
+//
+// metrics/assembly-metrics-config-files.adoc
+
+[id='ref-metrics-dashboards-{context}']
+= Example Grafana dashboards
+
+[role="_abstract"]
+If you deploy Prometheus to provide metrics,
+you can use Strimzi's example Grafana dashboards to monitor the following components:
+
+* Strimzi Operators
+* Kafka
+* Kafka ZooKeeper
+* Kafka Connect
+* Kafka MirrorMaker 2.0
+* Kafka Bridge
+* Cruise Control
+* Kafka Exporter
+
+All dashboards provide JVM metrics, as well as metrics specific to the component.
+For example, the Grafana dashboard for Strimzi Operators provides information on the number of reconciliations or custom resources they are processing.
+
+The dashboards are provided in the `examples/metrics/grafana-dashboards` directory of xref:deploy-examples-{context}[the example files provided with Strimzi].
+
+The example dashboards don't show all the metrics supported by Kafka.
+The dashboards are populated with a representative set of metrics for monitoring (as listed here).
+
+.Strimzi Operators dashboard metrics
```

Comment:
I'm still not sold on the value of this list. It is hard to maintain - #5583 changes the Connect dashboard. Do you have those changes already in? Will it need to be updated later? But also, we sell the metrics configuration and dashboards as examples which users can customize and which do not cover all available metrics. This is explained just above the list. But what sense does it make to list in detail all the metrics in the dashboards when they are just examples and do not cover all available metrics?

I think this just confuses those who miss the sentences above and is not really useful for those who read them because it has very little value.

Reply:
I wasn't entirely sure about continuing to list them because of the maintenance. Does it help people to know what's in the example config to begin with?  Maybe the json descriptions should be refreshed instead. I don't think the metrics are all described in the json. 

In the end, I've archived the descriptions and created a table to list the dashboards to match the _Example Prometheus metrics configuration_ section .